### PR TITLE
Fix-role-id

### PR DESCRIPTION
**Pull request by darmhoo**

### CODE REVIEW

File: src/utilities/order.js [hunk 1]
```diff
@@ -186,7 +186,8 @@ class Order {
     }
 
     setDiscount() {
-        this._discount = this.order.discount.name + " " + this.order.discount.percentage_discount;
+        // this._discount = this.order.discount.name + " " + this.order.discount.percentage_discount;
```
Comment (teebarg) [inline src/utilities/order.js:189]: why do we have this here?

Comment (darmhoo) [reply src/utilities/order.js:189]: it was a mistake

File: src/utilities/order.js [hunk 2]
```diff
@@ -143,7 +143,7 @@ class Order {
 
         /*discount amount*/
         let mFactor = this.count === 6 ? 0.5 : 1,
-            {percentage_discount: discount} = this.order.discount,
+            discount = this.order.discount,
```
Comment (teebarg) [inline src/utilities/order.js:146]: is this suppose to be part of this PR?

Comment (darmhoo) [reply src/utilities/order.js:146]: removed

**Review by teebarg — APPROVED** [review]

**Pull request merged**